src/js/components/FormField/doc.js

```diff
@@ -55,6 +55,16 @@ export const doc = FormField => {
         message: PropTypes.string,
       }),
       PropTypes.func,
+      PropTypes.arrayOf(
+        PropTypes.oneOfType([
+          PropTypes.shape({
+            regexp: PropTypes.object, // regular expression
+            validate: PropTypes.func,
+            message: PropTypes.string,
+          }),
+          PropTypes.func,
+        ]),
+      ),
```

Comment:
the description should include an array (not just a function or a regexp) line 69

Reply:
yep